examples/nextjs/src/utils/publicImageLoader.js

```diff
@@ -0,0 +1,11 @@
+"use client";
+
+import { useEffect, useState } from "react";
+
+const PublicImageLoader = ({ src }) => {
+    const [imageSrc, setImageSrc] = useState(null);
+    useEffect(() => setImageSrc(`${window.location.origin}${src}`), [src]);
+    return imageSrc ?? "/";
+};
+
+export default PublicImageLoader;
```

Comment:
Not sure if we should add this to the example

cc: @fmontes @rjvelazco

Reply:
No, this should be a basic example